.github/workflows/cdkactions_build-and-publish.yaml

```diff
@@ -13,8 +13,6 @@ jobs:
     strategy:
       matrix:
         python-version:
-          - 3.6
```

Comment:
Instead of modifying the YAML directly, it may be worth to also make a PR to update the Typescript in `kraken` that generates this YAML segment (even if we are planning to retire the TS in the long run).

Reply:
Oh shoot, this is on Armaan's repo